pkg/apiserver/installer/installer.go

```diff
@@ -221,7 +221,10 @@ func addObjectParams(ws *restful.WebService, route *restful.RouteBuilder, obj in
 				if docable, ok := obj.(documentable); ok {
 					desc = docable.SwaggerDoc()[jsonName]
 				}
-				route.Param(ws.QueryParameter(jsonName, desc).DataType(typeToJSON(sf.Type.String())))
+
+				if route.ParameterNamed(jsonName) == nil {
+					route.Param(ws.QueryParameter(jsonName, desc).DataType(typeToJSON(sf.Type.String())))
+				}
```

Comment:
The default list options and `MetricListOptions` have a `LabelSelector` property https://github.com/kubernetes/metrics/blob/v0.17.3/pkg/apis/custom_metrics/v1beta1/types.go#L77 and adding both causes an error when building the open api spec.

`Failed to build open api spec for root: duplicate parameter labelSelector for route [{},{},{},{},{},{},{},{},{},{},{},{},{},{}]`